File: Tareas/Unidad2/a2253332351_practica02/20_KmAMillas.java
import java.util.Scanner;

public class KmAMillas {
     public static void main(String[] args) {
         Scanner entrada = new Scanner(System.in);
         double kilometros, millas;

          System.out.println("Ingrese la distancia en kilómetros: ");
          kilometros = entrada.nextDouble();

          millas = kilometros * 0.621371;

          System.out.println("La distancia equivalente en millas es: " + millas);
     }
}

File: Tareas/Unidad2/a2253332351_practica02/20_KmAMillas.cpp
#include <iostream>
using namespace std;

int main() {
    double kilometros;
    double millas;

    cout << "Ingrese la distancia en kilómetros: ";
    cin >> kilometros;

    millas = kilometros * 0.621371;

    cout << "La distancia equivalente en millas es: " << millas << endl;

    return 0;
}
